### PR TITLE
Upload images &fix links in documentation

### PR DESCRIPTION
**Pull request by moishce**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/24871

## Description
Upload images for use in documentation

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
@moishce Why the hold up?

**Comment by ShahafBenYakir** [comment]
@moishce Please solve conflicts and check why build fails

**Comment by ShahafBenYakir** [comment]
@hod-alpert Do we need to keep the images in the repo? Isn't it enough to just change the URL in the readme as @moishce did?

**Comment by hod-alpert** [comment]
Also there is a folder for all images in content- `docs/images/Integrations`
The images should be there.
(And don't forget to update the commit hash again when you finish updating)

**Comment by moishce** [comment]
@glicht Where is the right place to put the image files in content- docs/images/Integrations? Or in the peck?

**Comment by glicht** [comment]
@moishce if you need to download and add the images create a doc_files dir in the integration directory.  See here: https://xsoar.pan.dev/docs/integrations/integration-docs#images

File: Packs/ThreatQ/ReleaseNotes/1_0_2.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+
+#### Integrations
+- fix images on __README.md__
```
Comment (ShahafBenYakir) [inline Packs/ThreatQ/ReleaseNotes/1_0_2.md:3]:
```suggestion
- __README.md__
Fixed images path in the documentation. 
```

File: Packs/Cybereason/ReleaseNotes/1_0_2.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+- __README.md__
+Fixed image path in documentation.
```
Comment (ShahafBenYakir) [inline Packs/Cybereason/ReleaseNotes/1_0_2.md:4]:
```suggestion
Fixed image path in the documentation.
```

**Comment by Itay4** [comment]
@moishce no need for release notes here, lets remove all release notes and merge

**Comment by ShahafBenYakir** [comment]
@Itay4 Removing RN will fail build and hooks and force merge will be needed.

**Comment by Itay4** [comment]
build fails on incorrect validations
merging

**Pull request merged**